src/api/questions.js

```diff
@@ -137,17 +137,21 @@ router.post(
       }
     }
 
-    const question = Question.build({
-      name: data.name,
-      // Questions in fixed-location queues should never have a location
-      location: res.locals.queue.fixedLocation ? '' : data.location,
-      topic: data.topic,
-      enqueueTime: new Date(),
-      queueId,
-      askedById: askerId,
-    })
+    const question = await Question.create(
```

Comment:
Can you clarify why there changes were necessary?

Reply:
We expect that the object returned from `Question.build` contains the appropriate askedBy model through association, but it doesn't do that anymore.
The [docs](http://docs.sequelizejs.com/manual/associations.html#creating-with-associations) suggest to use `.create` to create with association.

Reply:
Looking through the docs some more, it's possible to use `.build()` to achieve the same effect, but since we're not doing anything special between `.build()` and `.save()`, I don't see a reason to use it over `.create()`